PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -825,18 +827,20 @@ local function CreateCamera()
 			local finalConstant = 0
 
 			if lastThumbstickRotate then
+				local accelTime = UserInputService.VREnabled and vrNumOfSeconds or numOfSeconds
+				local actualMaxSpeed = UserInputService.VREnabled and vrMaxSpeed or maxSpeed
 				local elapsedTime = (currentTime - lastThumbstickRotate) * 10
-				currentSpeed = currentSpeed + (maxSpeed * ((elapsedTime*elapsedTime)/numOfSeconds))
+				currentSpeed = currentSpeed + (actualMaxSpeed * ((elapsedTime*elapsedTime)/accelTime))
 
-				if currentSpeed > maxSpeed then currentSpeed = maxSpeed end
+				if currentSpeed > actualMaxSpeed then currentSpeed = actualMaxSpeed end
 
 				if lastVelocity then
 					local velocity = (gamepadPan - lastThumbstickPos)/(currentTime - lastThumbstickRotate)
 					local velocityDeltaMag = (velocity - lastVelocity).magnitude
 
 					if velocityDeltaMag > 12 then
 						currentSpeed = currentSpeed * (20/velocityDeltaMag)
-						if currentSpeed > maxSpeed then currentSpeed = maxSpeed end
+						if currentSpeed > actualMaxSpeed then currentSpeed = actualMaxSpeed end
 					end
 				end
 
```

Comment:
I considered it and then assumed you'd probably prefer it this way. I guess my prediction was wrong.